packages/jest-haste-map/src/worker.ts

```diff
@@ -37,15 +34,14 @@ export async function worker(data: WorkerMessage): Promise<WorkerMetadata> {
       throw new Error('jest-haste-map: hasteImplModulePath changed');
     }
     hasteImplModulePath = data.hasteImplModulePath;
-    // $FlowFixMe: dynamic require
-    hasteImpl = (require(hasteImplModulePath): HasteImpl);
+    hasteImpl = require(hasteImplModulePath);
   }
 
-  let content;
-  let dependencies;
-  let id;
-  let module;
-  let sha1;
+  let content: string | undefined;
+  let dependencies: WorkerMetadata['dependencies'];
+  let id: WorkerMetadata['id'];
+  let module: WorkerMetadata['module'];
+  let sha1: WorkerMetadata['sha1'];
```

Comment:
in theory not needed, but it was way easier to track down where the assignment was wrong that it telling med the return at the bottom was wrong